content/goldcard-holders-faq/tax.md

```diff
@@ -63,3 +63,18 @@ The National Taxation Bureau provides free professional taxation consultation se
 During "tax season" there are dedicated English language counters for tax filing at
 National Taxation Bureau offices in major cities.
 
+## My salary is paid by an overseas company, where do I pay tax?
+Taiwan, and sometimes also in the overseas country.
```

Comment:
Perhaps using a word "in general". DTA agreements clarify definition of tax residency and how to resolve it if there's a conflict.